### ISSUE TITLE
Could you provide an example of a 70B model training config?

### ISSUE THREAD

**Issue by accupham** [tdrussell/qlora-pipe#1]
You mentioned it's possible to do a QLoRA training run on only 2x 3090. I have 4x, but I always OOM right before the first eval. VRAM is 20GB per GPU right before running eval, then tips over 23.69 GB right after. Could you provide a sample config toml and deepspeed json that can do a 70B model?

For reference, here is what I'm trying:

```bash
deepspeed --num_gpus=4 train.py --deepspeed --deepspeed_config examples/ds_config_7b.json --config examples/config_7b.toml
```
The only difference is that I replaced `model` with a 70B one (152334H/miqu-1-70b-sf).

**Comment by accupham** [comment]
Looks like setting `pipeline_stages = 4` seems to do the trick. If my config is successful i'll make a PR.

**Comment by kallewoof** [comment]
Yeah, this seems completely opposite of the description of pipeline_stages. With it set to 1 I was OOM'ing and bumping it to 2 (= GPUs) I was able to load models into VRAM with room to spare.

**Comment by tdrussell** [comment]
Hmm, is this the description that confusing?

> For example with 4 GPUs, and pipeline_stages=2, there are two data-parallel instances of the model, each divided across 2 GPUs

The pipeline_stages=2 corresponds to the "divided across 2 GPUs" part. But I see how the English here is confusing. And it being 4 / 2 = 2 doesn't help either. I can try to reword this part.

**Comment by kallewoof** [comment]
The way it is worded now to me looks like it will increase the VRAM requirement the higher the pipeline_stages value is but in reality the VRAM usage decreased (no OOM).
